### DISCUSSION TITLE
New Notifier Design?

### DISCUSSION THREAD

**Discussion by tcf1** [wokhansoft/WFN#123] [General]
I've been a lurker silently using WFN for quite some time and i'm pretty happy to see new releases. So much so i decided to create a github account to comment on the new Notifier.

While i appreciate the new Notifier not showing UAC prompts constantly, i don't understand why there is a separate widget that that now is on the desktop and what its purpose is. 

I also don't seem to get notified of new activity, in that they don't appear until i click the icon and then i can go through them.

Is it possible to get the both of both worlds where we can keep the new non-UAC notifier but just have it normally minimized near the clock until a new notification appears ?

**Comment by ExaFlop1** [comment]
I want to hide WFN taskbar icon in the tray.
